src/ffs/normalizer/basic.ts

```diff
@@ -0,0 +1,36 @@
+import { CID, FileContent } from '../../ipfs'
+import { TreeData, PrivateTreeData, Metadata} from '../types'
+import util from './util'
```

Comment:
Do you think it makes sense to rename these `util` module to `common`?
And then rename `common/index.ts` to `common.ts`?

I think it makes more sense here.
Seeing that `normalizer/basic.ts` and `normalizer/nested.ts` have these things "in common".

Reply:
Yeah I actually changed the naming of these around in the branch that I just merged into this one.
There's now `basic`, `nested`, `v0_0_0` and `v_1_0_0`, no more `util`.
The renaming is basically:
- `util -> basic`
- `basic -> v0_0_0`
- `nested` split into `nested` (utility functions) & `v_1_0_0` (exposed functions that get called by the lib)

Not totally sure how I feel about naming files after the versions :thinking:, but I also think it helps make it very explicit how each version of the file system operates. And we hopefully won't have _that_ many versions. Thoughts?

Reply:
> And then rename common/index.ts to common.ts

Yeah I wasn't sure if I would end up having more commons functions that I'd want to structure a little better. But agreed that this is unnecessary right now. I'll change it for now and we can always change back if it becomes an issue

Reply:
> but I also think it helps make it very explicit how each version of the file system operates
Agreed, I like the versioned modules. We shouldn't have that many versions anyway, and if we do, we can refactor then.